docs/introduction/best-practices.md

```diff
@@ -71,51 +71,48 @@ purpose of using A-Frame.
 [geometrymerger]: https://www.npmjs.com/package/aframe-geometry-merger-component
 
 
-Performance is critical in VR. A high framerate must be maintained in order for
-people to feel comfortable. Here are some ways to help improve performance of
-an A-Frame scene:
-
-- Use [recommended hardware specifications][hardware].
-- Use the **[stats component][stats]** to keep an eye on various metrics (FPS,
-  vertex and face count, geometry and material count, draw calls, number of
-  entities). We want to maximize FPS and minimize everything else.
+Performance is critical in VR. People feel comfortable when you maintain a high frame rate.
```

Comment:
keeping frame right high is a `must`. The re-wording makes it look less important

Reply:
```suggestion
Performance is critical in VR. To make people feel comfortable, you must maintain a high frame rate.
```

Reply:
Or even shorter (with or without bold formatting):

```suggestion
Performance is critical in VR. You **must** maintain a high frame rate.
```